lxd/storage/drivers/driver_pure.go

```diff
@@ -102,9 +120,9 @@ func (d *pure) Info() Info {
 
 // FillConfig populates the storage pool's configuration file with the default values.
 func (d *pure) FillConfig() error {
-	// Use iSCSI by default.
+	// Use NVMe by default.
```

Comment:
ah ok so we dont want to change history inside a single PR, so we need to rebase this, otherwise the reviewer has to keep track of how the commits change each file over time. Better to present the final version inside the PR.